web/src/components/zonemap.jsx

```diff
@@ -94,7 +102,10 @@ const ZoneMap = ({
         features,
       },
     };
-  }, [zones, selectedTimeAggregate]);
+    // TODO: `zoneValues` will change even in cases where the geometry doesn't change.
+    // This will cause this memo to re-update although it should only update when the
+    // geometry changes. This will slow down the map render..
+  }, [zoneValues, selectedTimeAggregate]);
```

Comment:
A performance optimisation is possible, see https://github.com/electricitymap/electricitymap-contrib/issues/4423